README.md

```diff
@@ -1,15 +1,18 @@
 # pyct
 
-A utility package that when installed makes various commands available
-to other packages. (Currently no sophisticated plugin system, just a
-try import/except in the other packages.) The same commands are
-available from within python. Can either add new subcommands to an
-existing argparse based command if the module has an existing command,
-or create the entire command if the module has no existing command.
-
-Currently, there are commands for:
-
-  * Copying examples and fetching data
+A utility package that:
+
+  1. makes various commands available to other packages. (Currently no
+     sophisticated plugin system, just a try import/except in the
+     other packages.) The same commands are available from within
+     python. Can either add new subcommands to an existing argparse
+     based command if the module has an existing command, or create
+     the entire command if the module has no existing
+     command. Currently, there are commands for copying examples and
+     fetching data.
+
+  2. provides various commands to help package building
```

Comment:
provides various commands to help package building, primarily as a convenience for project maintainers.

Reply:
Thanks. I took the prompt to improve the README a little more.